interfaces/Portalicious/src/app/components/page-layout/page-layout.component.html

```diff
@@ -4,7 +4,14 @@
     @if (projectId()) {
       <app-project-header [projectId]="projectId()!" />
     }
-    <div class="px-6 py-4 lg:px-24">
+    <div class="px-6 pb-8 lg:px-24">
+      <div class="flex items-end justify-between py-9">
+        <h1 class="my-0 min-w-48 text-3xl">
+          {{ title() }}
+        </h1>
```

Comment:
We should remove the `<h1>`-element from the `<app-header>`-component; As there should only be one of those.

Reply:
I will sneak it into #5697